src/main/java/Duke.java

```diff
@@ -1,10 +1,50 @@
+import java.util.*;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String logo = "____________________________________________________________\n"
+                + " Hello! I'm Duke\n"
+                + " What can I do for you?\n\n"
+                +"____________________________________________________________\n";
+        System.out.println(logo);
+        Scanner stringScanner = new Scanner(System.in);
+        String userQuery = stringScanner.nextLine();
+        String userQueryReturn = "";
+        ArrayList<Task> taskItems = new ArrayList<Task>();
+        int itemCount = 0;
+        while(!userQuery.equalsIgnoreCase("bye")){
+            if(userQuery.contains("done")){
+                int doneIndex = Integer.parseInt(userQuery.substring(5));
+                taskItems.get(doneIndex-1).setStatus(true);
+                userQueryReturn = "____________________________________________________________\n"
+                        +" Nice! I've marked this task as done: \n"
+                        + " ["+taskItems.get(doneIndex-1).getTypeIcon()+"]["+taskItems.get(doneIndex-1).getStatusIcon()+"] "+taskItems.get(doneIndex-1).getDescription()+"\n"
+                        + "____________________________________________________________\n";
```

Comment:
Maybe these borders could be declared as a constant at the start, to be used throughout?
Perhaps these variables can be broken up into intermediate ones to improve readability?

[https://nus-cs2113-ay2021s2.github.io/website/schedule/week4/topics.html#implementation-code-quality-readability-basic-avoid-complicated-expressions](url)